contributors/devel/sig-architecture/api_changes.md

```diff
@@ -1253,7 +1260,12 @@ This ensures a cluster with multiple servers at skewed releases (which happens d
 will not allow data to be persisted which the previous release of the API server would choke on.
 
 Typically, a feature gate is used to do this rollout, starting in alpha and disabled by default in release 1,
-and graduating to beta and enabled by default in release 2.
+and graduating to stable and enabled by default in release 2.
+With beta as an intermediate step, release 3 (stable) would enable the feature gate by default.
+The behavior changed with Kubernetes v1.24 
+(for details see the [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/3136-beta-apis-off-by-default/README.md)).
+Beta APIs introduced before v1.24 will be enabled by default.
+Beta API introduced after v1.24 will be disabled by default.
```

Comment:
drop this change, this section of the doc is not referring to a new beta API version/endpoint

Reply:
Dropped.